includes/EmailNotification.php

```diff
@@ -465,29 +566,29 @@ private function getEmailExtensionController() {
 		$controller = false;
 
```

Comment:
Had a hard time finding class usage because all of these were strings; PHPStorm had no idea they were being accessed.  Does not contribute to logging but a good thing to update.